test/Datadog.Trace.ClrProfiler.IntegrationTests/LogsInjectionTestBase.cs

```diff
@@ -0,0 +1,161 @@
+// <copyright file="LogsInjectionTestBase.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public abstract class LogsInjectionTestBase : TestHelper
+    {
+        private readonly string _excludeMessagePrefix = "[ExcludeMessage]";
+
+        public LogsInjectionTestBase(ITestOutputHelper output, string sampleName)
+            : base(
+                new EnvironmentHelper(
+                    sampleName: sampleName,
+                    typeof(TestHelper),
+                    output,
+                    prependSamplesToAppName: false),
+                output)
+        {
+        }
+
+        public enum UnTracedLogTypes
+        {
+            /// <summary>
+            /// UnTraced logs do not include any dd_ properties
+            /// </summary>
+            None,
+
+            /// <summary>
+            /// UnTraced logs include the dd_ properties, but without any values
+            /// </summary>
+            EmptyProperties,
+
+            /// <summary>
+            /// UnTraced logs include dd_service, dd_env, and dd_version with their correct values
+            /// but no dd_trace_id
+            /// </summary>
+            TWOLPropertiesOnly,
```

Comment:
nit: I thought this was "Unified Service Tagging", not "Tracing Without Limits"?

Reply:
Oops, you're right. Updated the name